portfolio/src/main/webapp/scripts/blog_script.js

```diff
@@ -66,19 +66,22 @@ function createCommentElement(comment, currUserEmail, isAdmin){
     return commentElement;
 }
 
-function deleteComment(comment){
-    const params = new URLSearchParams();
-    params.append('id', comment.id);
-    fetch('/delete-comment', {method: 'POST', body:params});
+async function deleteComment(comment){
+    (async () =>{
```

Comment:
Any reason to use IIFE? I feel it is same as 
```
function deleteComment(comment){
    const params = new URLSearchParams();
    params.append('commentId', comment.commentId);
    fetch('/delete-comment', {method: 'POST', body:params});
}
```

Reply:
Before using it, the code wasn't actually deleting my comments from the datastore, only the page view.

Reply:
Just FYI, you could use try
```async function deleteComment(comment){
    const params = new URLSearchParams();
    params.append('commentId', comment.commentId);
    await fetch('/delete-comment', {method: 'POST', body:params});
}
```
`await` will block until the fetch gets done.